include/emulator/errordisplay.h

```diff
@@ -0,0 +1,80 @@
+#ifndef _CODE_800633F8_H
+#define _CODE_800633F8_H
+
+#include "emulator/banner.h"
+#include "emulator/stringtable.h"
+#include "revolution/gx.h"
+#include "revolution/sc.h"
+#include "revolution/types.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define FADE_TIMER_MAX 15
+
+#define FLAG_COLOR_WHITE (0 << 0)
+#define FLAG_COLOR_YELLOW (1 << 0)
+#define FLAG_RESET_FADE_TIMER (1 << 1)
+
+typedef struct DisplayFiles {
+    /* 0x00 */ SCLanguage eLanguage;
+    /* 0x04 */ char* szErrors;
+    /* 0x08 */ char* szSaveComments;
```

Comment:
maybe `szErrorsFilename` and `szSaveCommentsFilename`? I initially though these were arrays of strings based on the name